### PR TITLE
Updated documentation: API contracts

### PR DESCRIPTION
**Pull request by joeldudleyr3**
Refreshes the Contracts API docs for Corda v1.0.

### CODE REVIEW

File: docs/source/api-contracts.rst [hunk 1]
```diff
@@ -197,98 +225,12 @@ execution of ``verify()``:
 
             @Override
             public void verify(LedgerTransaction tx) {
-                final CommandWithParties<Commands> command = requireSingleCommand(tx.getCommands(), Commands.class);
+                final CommandWithParties<Commands> command = tx.findCommand(Commands.class, cmd -> true);
```
Comment (szymonsztuka) [inline docs/source/api-contracts.rst]: This seems to be Kotlin ('cmd -> true') but the snippet is for Java?

Comment (joeldudleyr3) [reply docs/source/api-contracts.rst]: That's valid Java I think? It compiles at least.

Comment (szymonsztuka) [reply docs/source/api-contracts.rst]: Right, however unless I have different source version, shouldn't it be:   
final Command<Commands> command = tx.findCommand(Commands.class, cmd -> true);

**Comment by szymonsztuka** [comment]
Looks good.

**Comment by josecoll** [comment]
How about you give the commit header a sensible title?

**Pull request merged**